clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/deploy/DescriptionAuthorizer.java

```diff
@@ -110,7 +114,8 @@ public void authorize(T description, Errors errors) {
     if (description instanceof ResourcesNameable) {
       ResourcesNameable resourcesNameable = (ResourcesNameable) description;
 
-      if (!resourcesNameable.requiresAuthorization()) {
+      if (!resourcesNameable.requiresAuthorization()
+          || dynamicConfigService.isEnabled("aws.fiat.authorize.resources-nameable", false)) {
```

Comment:
I also added a (hopefully temporary) dynamic config here so we can quickly disable this if we need to.